src/pages/audio-rewards-page/components/modals/ChallengeRewards.tsx

```diff
@@ -253,11 +262,45 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
     }
   }, [dispatch, challenge, specifier])
 
+  const [hasClaimed, setHasClaimed] = useState(false)
+  const previouslyUndisbursedChallenges = useSelector(getUndisbursedChallenges)
+
   useEffect(() => {
```

Comment:
Could use a comment explaining what this is about, and probably extracting to it's own fn